docs/modules/ROOT/pages/release-notes/README.adoc

```diff
@@ -6,6 +6,9 @@ Detailed information about features and fixes in every release.
 [[all-release-notes]]
 == All Release Notes
 
+*2021*
+* 2021 January - xref:release-notes/release-notes-24-0.adoc[Payara Enterprise 5.24.0]
```

Comment:
Empty line required between `*2021*` and `* 2021 January ...` for proper rendering.